teku/src/main/java/tech/pegasys/teku/cli/converter/GraffitiConverter.java

```diff
@@ -16,22 +16,21 @@
 import java.nio.charset.StandardCharsets;
 import org.apache.tuweni.bytes.Bytes32;
 import picocli.CommandLine;
+import tech.pegasys.teku.validator.api.Bytes32Parser;
 
 public class GraffitiConverter implements CommandLine.ITypeConverter<Bytes32> {
   @Override
   public Bytes32 convert(final String value) {
     byte[] input = value.getBytes(StandardCharsets.UTF_8);
-    if (input.length > 32) {
+    try {
+      return Bytes32Parser.toBytes32(input);
```

Comment:
(nit) I think we can use your `String` -> `Bytes32` helper directly here:
```suggestion
      return Bytes32Parser.toBytes32(value);
```